sbncode/CAFMaker/cafmakerjob_sbnd.fcl

```diff
@@ -2,6 +2,7 @@
 
 #include "services_sbnd.fcl"
 #include "geometry_sbnd.fcl"
+#include "eventweight_genie3_sbn_A.fcl"
```

Comment:
Probably a different discussion, but I feel like we need to separate the concept of "CAF-stage reco" from the actual cafmaker fcl itself. At the moment if you just want to CAF something you by default get all this stuff.

Also, I don't like the way we do inheritance between job fcls. I think the module configurations/tables should inherit from each other, but then the final fcl you run should actually have a complete structure, with all the usual blocks, although they will be populated just by a `@table` from elsewhere.

You're just following what's already done, so it's nothing about this PR specifically.

Reply:
I agree some of this stuff should be moved to earlier stages. We are kinda stuck now with what is in the reco2 files we have but before the next production pushing some of this to earlier in the chain would be good. It would also help to keep the CAFMaker stage quick if we want to rerun it, quite a few of these things (e.g. the CRT matching) are almost certainly not going to change very often.